runtimerevolution/web3-roulette
# [WEB3-33] Details aside content

# Description

This PR updates the UI of the details page:
- QR code section and its options: download PDF and copy to clipboard
- Add winning chance near participants
- Refractor participate button and add it to details page as well
- Improve the PDF layout

Also included on this PR:
- Fix empty state verifications
- The next giveaway used for the countdown card no longer requires participating state, since the user can participate through details page

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-33

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Use case 1:
- Login as an admin user
- Go to the details page of some giveaway
- Verify the displayed information and the design
- Try to download the PDF
- Try to copy to clipboard

Use case 2:
- Login as a normal user
- Go to the details page of some giveaway
- Verify the displayed information and that the design does not contain QR code section
- Verify the participate button and the winning chance
- Try to participate

## Checklist:
